.github/scripts/run-tests.sh

```diff
@@ -29,24 +39,32 @@ print_failed_tests() {
   fi
 }
 
-run_tests action-token-authenticator -Pwildfly-managed
-run_tests action-token-required-action -Pwildfly-managed
+if [ ! -n "$PRODUCT" ] || [ "$PRODUCT" == "false" ]; then
+  run_tests action-token-authenticator -Pwildfly-managed
+  run_tests action-token-required-action -Pwildfly-managed
+fi
 
 . .github/scripts/export-keycloak-version.sh
-JS_VERSION_OPTION=""
 
 if [ -n "$KEYCLOAK_VERSION" ]; then
   JS_VERSION_OPTION="-Dkeycloak.js.version=$KEYCLOAK_VERSION"
+elif [ -n "$PRODUCT_VERSION" ]; then
+  JS_VERSION_OPTION="-Dkeycloak.js.version=$PRODUCT_VERSION"
+else
+  JS_VERSION_OPTION=""
 fi
 
+
 run_tests app-angular2 -Pwildfly-managed "$JS_VERSION_OPTION"
 
 # we need to run authz springboot tests first as they are the only ones relying on manual js-policies deployment
 # other tests deploy (and the removes) the policies automatically which then later removes even the manually deployed ones
 run_tests app-authz-rest-springboot -Pspring-boot
 run_tests app-authz-spring-security -Pspring-boot
 run_tests app-authz-springboot -Pspring-boot
-run_tests app-authz-springboot-multitenancy -Pspring-boot
+if [ ! -n "$PRODUCT" ] || [ "$PRODUCT" == "false" ]; then
```

Comment:
I don't like so many identical IFs here (`if [ ! -n "$PRODUCT" ] || [ "$PRODUCT" == "false" ]; then
`). :) Not sure what to do about this, though. Maybe create some list of excluded QS in prod and handle it in `run_tests`? Or at least group the excluded QS into a single IF?